js/options/options.js

```diff
@@ -335,114 +334,97 @@ let Options = (function(){
     }
 
     self.init = async function() {
-        let settings = SyncedStorage.load();
+        let settings = SyncedStorage.init();
         let currency = fetch(chrome.runtime.getURL('json/currency.json'))
```

Comment:
Refactor to use layer between fetch

Reply:
Created a new PR tfedor/AugmentedSteam#150 https://github.com/tfedor/AugmentedSteam/commit/45fb1cf600915364438babc221ac62d57032a1f3